### PR TITLE
chore(changelog): fix formatting for alpha.53

### PR DESCRIPTION
**Pull request by valorkin**
@vicb please check title I have added 
what should be there for real?

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -33,8 +33,16 @@
 
 ### BREAKING CHANGES
 
-* Before import {Observable} from 'angular2/core'
-After import {Observable} from 'rxjs/Observable';
+* `Observable` removed from `angular2/core` API
```
Comment (vicb) [inline CHANGELOG.md]: `Observable` are no more re-exported from `angular2/core`

**Comment by vicb** [comment]
Please also change your commit message to keep the headline only (& alfa -> alpha). Thanks.

**Comment by valorkin** [comment]
@vicb done

**Comment by vicb** [comment]
Thanks !

**Pull request closed without merging**